custom_components/alarms_and_reminders/sentence_manager.py

```diff
@@ -80,8 +80,8 @@ async def async_setup_sentence_files(hass: HomeAssistant) -> None:
                     lambda: target_dir.mkdir(parents=True, exist_ok=True)
                 )
 
-                # Write YAML file
-                target_file = target_dir / f"alarms_and_reminders_{sentence_type}.yaml"
+                # Write  file
+                target_file = target_dir / f"alarms&reminders_{sentence_type}.yaml"

```

Comment:
putting a `&` in a file name sounds unusual, I don't think it can cause issues though